doc/Custom.md

```diff
@@ -87,4 +87,6 @@ services:
       - ./platform-config.yaml:/opt/conduktor/platform-config.yaml
     environment:
       CDK_IN_CONF_FILE: /opt/conduktor/platform-config.yaml
+volumes:
```

Comment:
Can you put an empty line before this line please? 

It's kind of a convention in the Docker Compose world

Reply:
Done